### ISSUE TITLE
[SampleCard-Inputs; Config-cortanaskills-classic]: The color contrast ratio for the text is 1:1 which does not meet requirement of at least 4.5:1

### ISSUE THREAD

**Issue by carlos-zamora** [microsoft/AdaptiveCards#7400]
https://microsoft.visualstudio.com/OS/_workitems/edit/38021465
